packages/@lwc/template-compiler/src/__tests__/fixtures/scoped-slots/invalid/slot-data/name-is-expression/ast.json

```diff
@@ -1 +1,255 @@
-{}
\ No newline at end of file
+{
```

Comment:
Move this test case out of the `invalid` folder now that this usage is valid.

Reply:
This still needs to be addressed.

Reply:
Addressed.